apps/modernization-ui/src/apps/search/event/components/EventTypeAccordion/EventTypeAccordion.tsx

```diff
@@ -1,6 +1,6 @@
 import { Accordion } from '@trussworks/react-uswds';
 import { AccordionItemProps } from '@trussworks/react-uswds/lib/components/Accordion/Accordion';
-import { SelectInput } from 'components/FormInputs/SelectInput';
+import { SelectInput } from '../../../../../components/FormInputs/SelectInput';
```

Comment:
something thats optional here is to remove the extra ../../../../../ that prettier may set but this is a non blocking item